### DISCUSSION TITLE
musicblocks-2 renamed to musicblocks-v4

### DISCUSSION THREAD

**Discussion by meganindya** [sugarlabs/musicblocks-v4#38] [General]
The name of the next instalment of Music Blocks was initially thought to be _Music Blocks 2.0_. However, it is not a new application in free sense. To end users it'll still be Music Blocks, just a newer generation.

The current major version of Music Blocks is **v3**. Therefore, the new application will actually be a new major version release **v4**, which would be backwards incompatible. Any previous references to _Music Blocks 2.0_ will hereafter imply _Music Blocks v4_.

Hence, I've renamed the new repositories `musicblocks-2` and `musicblocks-lib` as **`musicblocks-v4`** and **`musicblocks-v4-lib`**, respectively. I'll admit I didn't think this one through before.

If you have forked any or both of these, please update the names in your account and local system. Also, update the remote upstream to the new URL. Or instead, you can just delete the existing ones and re-fork and re-clone.

**`musicblocks-v4`** will have the UI implementation of the **_Music Blocks_ (_v4_)** application for desktop web and eventually other platforms. **`music blocks-v4-lib`** will contain the core foundations, and will be bundled and interfaced with the code in the prior.

**Comment by pikurasa** [comment]
Makes sense to me. Agreed.
